create a new book list

FlorRodriguezA opened the pull request:
## Summary
- Used flexbox for all  book covers, and inside the book cover too 

## Screenshot
![Captura de pantalla de 2021-02-23 10-59-21](https://user-images.githubusercontent.com/49420114/108854133-4403b480-75c6-11eb-9f47-ec2d949a9c35.png)

## Trello Card
https://trello.com/c/3KkyuGEw

FlorRodriguezA committed:
create a new book list

FlorRodriguezA committed:
Add nav bar and logo

FlorRodriguezA commented:
Summary
Reduced the margin, add a nav bar in the top, and a the Logo Wolox

Screenshot
![Captura de pantalla de 2021-02-23 16-19-15](https://user-images.githubusercontent.com/49420114/108896368-96a79580-75f3-11eb-8b51-50b9c0493229.png)

Trello Card
https://trello.com/c/HUcyRij8

FlorRodriguezA committed:
sticky position for the bar

FlorRodriguezA commented:
Summary
I added the sticky position for the bar

Screenshot
![Captura de pantalla de 2021-02-23 16-44-19](https://user-images.githubusercontent.com/49420114/108898804-6ca3a280-75f6-11eb-8330-16bfbb5ea669.png)


Trello Card
https://trello.com/c/HUcyRij8

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,76 @@
+  margin-left: 120px;
+}
+
+.all-group {
```
I think there's a better name for this class

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,76 @@
+  background-color: $wild-sand;
+}
+
+.all-group2 {
```
Same for this, avoid using numbers in class names 😝

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,76 @@
+  align-items: center;
+}
+
+.group {
```
Group of what?

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,76 @@
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 36px;
+  margin-right: 36px;
```
Always use the `margin` property instead of specifying in multiple margin properties. For example, this can be:

`margin: 35px 35px 0 0;`

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,76 @@
+  line-height: 18px;
+  margin-top: 8px;
+  margin-right: 30px;
+}
```
Remember to add the EOF 😄

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,76 @@
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  margin-top: 12px;
+  margin-right: 30px;
```
Same here for the margins

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,76 @@
+  color: $gray;
+  font-size: 14px;
+  line-height: 18px;
+  margin-top: 8px;
+  margin-right: 30px;
```
Same here

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,76 @@
+
+.nav-bar {
+  height: 70px;
+  width: 1169px;
```
If you set this fixed size, the navbar will not properly adapt itself to other display sizes. What you can do is set it to:

`width: 100vw;` or `width: 100%;`

With the first one, you are setting it to be the width of the ViewWidth of the browser window. The second is telling the navbar to occupy the available space it has according to its parent

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,76 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+@import 'equalizer.scss';
+@import 'colors.scss';
+
+
+* {
+  font-family: Roboto, sans-serif;
+}
```
Move this to the `application.scss` since that's your main file 😄

mcallegari10 commented on listado-de-libros/index.html:
```diff
@@ -0,0 +1,66 @@
+
+  <body>
+    <div class="all-group">  
+      <div class="nav-bar">
```
This should be a `nav` tag, not a `div`

mcallegari10 replied:
And maybe this should be outside the main `div`

mcallegari10 reviewed (changes requested):
I dont see the detail view you were working on the main branch. Remember to do a `git pull` and then move to this branch and `git merge main`. That way you'll bring all your work into this one 😄

FlorRodriguezA committed:
Merge branch 'main' into book-list

FlorRodriguezA committed:
change styles and add logout

FlorRodriguezA commented:
Summary
Made all the requested changes. Add a blue bar on the top and a logout. Also changed flex styles for the covers
I forgot to delete line 10 in the html (class blue-nav-bar)

Screenshot
![Captura de pantalla de 2021-02-24 16-28-06](https://user-images.githubusercontent.com/49420114/109055679-40ebef80-76be-11eb-8b51-c49a76c71645.png)


Trello Card
https://trello.com/c/HUcyRij8

mcallegari10 commented on listado-de-libros/index.html:
```diff
@@ -3,58 +3,60 @@
-      <div class="all-group2">
-        <div class= group>
-          <img  class ="book-cover" src="book-cover.png" alt="Book cover">
+    <nav class="blue-nav-bar"></nav>
```
What's this `blue-nav-bar` for? 🤔

mcallegari10 commented on listado-de-libros/index.html:
```diff
@@ -3,58 +3,60 @@
+    <nav class="blue-nav-bar"></nav>
+      <nav class="nav-bar">
+          <img class="logo-wolox" src="LogoWolox.png" alt="Logo Wolox">
+          <a class="logout">Logoout</a>
```
```suggestion
          <a class="logout">Log out</a>
```

mcallegari10 replied:
BTW, I think this is not part of the `navbar`

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -1,42 +1,44 @@
-
 * {
   font-family: Roboto, sans-serif;
 }
```
Move this to the `application.scss` too 😉

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -1,42 +1,44 @@
+  font-size: 18px;
+  line-height: 21px;
+  position: absolute;
+  margin: 26px 0 0 600px;
```
`600px` whaaaaaaat 💥 . Remember not to position elements with margin 😄

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -1,42 +1,44 @@
+  color: $black;
+  font-size: 18px;
+  line-height: 21px;
+  position: absolute;
```
I think this should not be `absolute` 🤔

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -1,42 +1,44 @@
-.all-group2 {
+.all-book-cover {
+  height: 768px;
+  width: 100%;
```
The proper way to set a width is:

```css
.my-class {
  ...
  max-width: <pixels>px;
  width: 100%;
  ...
}
```

FlorRodriguezA committed:
Changes to the nav bar

FlorRodriguezA commented:
Summary
Used Flex to modify the position of the elements of the nav bar. 
Used top and right to position them correctly, with relative position

Screenshot
![Captura de pantalla de 2021-02-25 09-00-41](https://user-images.githubusercontent.com/49420114/109150724-06c73000-7748-11eb-845c-ffe3e9de2ca8.png)

Trello Card
https://trello.com/c/HUcyRij8

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,75 @@
+.nav-bar {
```
I think the `nav` classes can be moved to their own file 😄 . And BTW, this is the `detail` or `book-list` SASS file?

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,75 @@
+  font-size: 18px;
+  line-height: 21px;
+  position: relative;
+  top: 20px;
```
If the position is `relative`, you won't need this anymore

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,75 @@
+.nav-bar {
+  height: 70px;
+  width: 100vw;
+  max-width: 1300px;
```
I think this max is not necessary 🤔

mcallegari10 commented on listado-de-libros/scss/detail.scss:
```diff
@@ -0,0 +1,75 @@
+  height: 70px;
+  width: 210px;
+  position: relative;
+  right: 50px;
```
I think this should be a `nav-bar` padding

FlorRodriguezA committed:
New scss, and style changes

FlorRodriguezA committed:
New title

FlorRodriguezA commented:
Summary
Create a new Scss (nav-bar.scss). And I fixed the errors.
And I changed the title


Screenshot
![Captura de pantalla de 2021-02-25 15-45-48](https://user-images.githubusercontent.com/49420114/109201503-ae605480-7780-11eb-8f0d-dcfe105102ad.png)


Trello Card
https://trello.com/c/HUcyRij8

mcallegari10 commented on listado-de-libros/scss/nav-bar.scss:
```diff
@@ -0,0 +1,26 @@
+  line-height: 21px;
+  position: relative;
+  padding-top: 20px;
+}
```
Remember the EOF line 😄

mcallegari10 reviewed (approved):
Remember to center the list and ready to go 🚀

FlorRodriguezA committed:
style change in the position of elements

FlorRodriguezA committed:
delete height

mcallegari10 reviewed (approved):
![](https://media.giphy.com/media/l1KsJua7qIQnevGyk/giphy.gif)
